pom.xml

```diff
@@ -171,6 +186,12 @@ under the License.
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.eclipse.sisu</groupId>
+        <artifactId>sisu-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
```

Comment:
I think it's better to add `org.eclipse.sisu:org.eclipse.sisu.inject:0.3.5` as a compile time dependencies to leverage the annotation processor instead of using a plugin.

Reply:
Am on the edge here. If we start doing that, sisu will be "yet another" plexus-utils that are "auto-dragged" by any plugin, no? sisu plugin does it's job at build time and makes project dependency less to pull....